.github/workflows/test.yml

```diff
@@ -6,10 +6,10 @@ name: Tests
 on:
```

Comment:
All the changes in this file should not be part of this PR. Please remove them and let this PR only focus on the role and permission resource.